lib/galaxy/web/framework/decorators.py

```diff
@@ -326,7 +326,7 @@ def format_return_as_json(rval, jsonp_callback=None, pretty=False):
 
     Use `pretty=True` to return pretty printed json.
     """
-    dumps_kwargs = dict(indent=4, sort_keys=True) if pretty else {}
+    dumps_kwargs = dict(indent=4) if pretty else {}
```

Comment:
I don't think we should do this until we sunset python 3.5. You could add the index of the element [here](https://github.com/mvdbeek/galaxy/blob/dev/lib/galaxy/tool_util/parser/xml.py#L601) and then sort the test elements where needed. Do you maybe want to just the xsd changes in for 20.05 ?

Reply:
Your are probably right -- even if it seems safe in most places where `format_return_as_json` is used (pretty is either set to `False` or set to `True` only in debug mode). 

The only place where its `True` is here: 

https://github.com/galaxyproject/galaxy/blob/7eb2fcb708e7b63e17800c87613ddfa5497c0654/lib/galaxy/webapps/galaxy/api/workflows.py#L485

An alternative might be to add another parameter (e.g. `sort_keys=False`) to `format_return_as_json`. Tell me your preference and I will go ahead :)

Reply:
I'd go with the xsd changes only, but if you want to dig deeper I think that adding the index of the element to the test definition seems more solid to me.